files/initramfs-tools/union-mount.j2

```diff
@@ -11,17 +11,37 @@ case $1 in
     ;;
 esac
 
+docker_inram=false
+logs_inram=false
+secureboot=false
+bootloader=generic
+
+# Extract kernel parameters
+for x in $(cat /proc/cmdline); do
+    case "$x" in
+        Aboot=*)
+            bootloader=aboot
+            ;;
+        docker_inram=on)
+            docker_inram=true
+            ;;
+        logs_inram=on)
+            logs_inram=true
+            ;;
+        secureboot=on)
```

Comment:
Could you please merge to the latest master branch and change secure_boot_enable to use secureboot?

Reply:
We prefer PR commits rebase frequently onto master than merge from master, if possible.

---
In reply to: [440557530](https://github.com/Azure/sonic-buildimage/pull/4741#discussion_r440557530) [](ancestors = 440557530)

Reply:
Sure I will rebase my change. I tend to prefer rebase over merge anyways but wasn't sure what the preferred approach was there. I'll cleanup my branch.